onnxruntime/core/session/IOBinding.cc

```diff
@@ -109,6 +109,8 @@ void IOBinding::ClearOutputs() {
 
 const std::vector<std::string>& IOBinding::GetOutputNames() const { return output_names_; }
 
+const std::vector<OrtValue>& IOBinding::GetOutputs() const { return outputs_; }
+
 std::vector<OrtValue>& IOBinding::GetOutputs() { return outputs_; }
```

Comment:
do we need the method returning non-const outputs in the internal IOBinding class ? Under what circumstances do we need these two variants ? A comment may help someone trying to understand the code..

Reply:
It is an overload.